apps/api-gateway/src/tests/rate-limit.test.ts

```
@@ -0,0 +1,76 @@
+    
+    fastifyMock = {
+      register: jest.fn(),
+    } as unknown as FastifyInstance;
+  });
+  
+  afterEach(() => {
+    jest.clearAllMocks();
+  });
+  
+  it('should use default values when environment variables are not set', async () => {
+    delete process.env.RATE_LIMIT_MAX;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)